server_utils/origin.go

```diff
@@ -492,7 +492,8 @@ from S3 service URL. In this configuration, objects can be accessed at /federati
 	return originExports, nil
 }
 
-func CheckSentinelLocation(exports []OriginExport) (ok bool, err error) {
+// Check the sentinel files from Origin.Exports
+func CheckOriginSentinelLocations(exports []OriginExport) (ok bool, err error) {
```

Comment:
```suggestion
func CheckOriginSentinelLocation(exports []OriginExport) (ok bool, err error) {
```

Reply:
The `s` was intentional as we are checking multiple locations for different prefixes